docs/developing.rst

```diff
@@ -0,0 +1,199 @@
+====================
+Developer Guidelines
+====================
+
+This webpage is intended to guide users through making making changes to
+``tedana``'s codebase, in particular working with tests.
+The worked example also offers some guidelines on approaching testing when
+adding new functions.
+
+Adding and Modifying Tests
+==========================
+Testing is an important component of development.
+For simplicity, we have migrated all tests to ``pytest``.
+There are two basic kinds of tests: unit and integration tests.
+Unit tests focus on testing individual functions, whereas integration tests focus on making sure
+that the whole workflow runs correctly.
+
+For unit tests, we try to keep tests on the same module grouped into one file.
```

Comment:
```suggestion
For unit tests, we try to keep tests from the same module grouped into one file.
```

Reply:
I'd add a subheading to this to specify that it's unit testing (as opposed to other cases outlined below)